src/courses/templates/courses/mark_calculations.html

```
@@ -40,13 +42,28 @@
         {% endfor %}
       </table>
+      {% if no_course_uses_marks %}
+        <p>
+          {% if num_courses == 1 %}Your course runs{% else %}Your courses run{% endif %} on XP alone, so
+          there is no percentage to work out. Keep earning XP.
+        </p>
+      {% else %}
       <h3>Explanation</h3>
       <p>
         Here is an explanation of how I calculated your mark. If you have any questions about this please ask!
+        {% if num_courses > 1 and obj.course != courses.first.course %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)